exporter/signalfxexporter/correlation/correlation.go

```diff
@@ -105,13 +105,6 @@ func (cor *Tracker) AddSpans(ctx context.Context, traces pdata.Traces) error {
 			return
 		}
 
-		var err error
-		cor.correlation, err = newCorrelationClient(cor.cfg, cor.accessToken, cor.params)
```

Comment:
I see that `cor.correlation` is used in the `cor.traceTracker = tracetracker.New(...)` call. Is there another place where `cor.correlation` initialized the call?

Reply:
Yes, it has been moved to L133 as part of Tracker.Start(...). https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/3511/files#diff-c5e5fb87029d1d0003cb0256db0ccfbc8e913b3747d4869fed0d23332a7a8b8dR132

Reply:
oh, sorry, missed that line